packages/react/src/components/Breadcrumb/Breadcrumb.base.tsx

```diff
@@ -47,131 +41,103 @@ const nonActionableItemProps: Partial<IContextualMenuItemProps> = {
+    theme: theme!,
+  });
 
-    renderedOverflowItems = [...renderedOverflowItems, movedItem];
+  const nativeProps = getNativeProps<React.HTMLAttributes<HTMLDivElement>>(props, htmlElementProperties, ['className']);
 
-    return { ...data, renderedItems, renderedOverflowItems };
+  const onBreadcrumbClicked = (ev: React.MouseEvent<HTMLElement>, item: IBreadcrumbItem) => {
+    item.onClick?.(ev, item);
   };
 
-  /**
-   * Remove the last item of the overflow set and insert the item as the start of the rendered set past the overflow
-   * point.
-   */
-  private _onGrowData = (data: IBreadcrumbData): IBreadcrumbData | undefined => {
-    let { renderedItems, renderedOverflowItems } = data;
-    const { overflowIndex, maxDisplayedItems } = data.props;
-
-    renderedOverflowItems = [...renderedOverflowItems];
-    const movedItem = renderedOverflowItems.pop();
-
-    if (!movedItem || renderedItems.length >= maxDisplayedItems!) {
-      return undefined;
+  const renderItem = (item: IBreadcrumbItem) => {
+    if (item.onClick || item.href) {
+      return (
+        <Link
+          as={item.as}
+          className={classNames.itemLink}
+          href={item.href}
+          aria-current={item.isCurrentItem ? 'page' : undefined}
+          // eslint-disable-next-line react/jsx-no-bind
+          onClick={ev => onBreadcrumbClicked(ev, item)}
+          role={item.role}
+        >
+          <TooltipHost content={item.text} overflowMode={TooltipOverflowMode.Parent} {...tooltipHostProps}>
+            {item.text}
+          </TooltipHost>
+        </Link>
+      );
+    } else {
+      const Tag = item.as || 'span';
+      return (
+        <Tag className={classNames.item}>
+          <TooltipHost content={item.text} overflowMode={TooltipOverflowMode.Parent} {...tooltipHostProps}>
+            {item.text}
+          </TooltipHost>
+        </Tag>
+      );
     }
-
-    renderedItems = [...renderedItems];
-    renderedItems.splice(overflowIndex!, 0, movedItem);
-
-    return { ...data, renderedItems, renderedOverflowItems };
   };
 
-  private _onRenderBreadcrumb = (data: IBreadcrumbData) => {
+  const renderBreadcrumb = (data: IBreadcrumbData) => {
     const {
       ariaLabel,
       dividerAs: DividerType = Icon as React.ElementType<IDividerAsProps>,
-      onRenderItem = this._onRenderItem,
+      onRenderItem = renderItem,
       overflowAriaLabel,
-      overflowIndex,
       onRenderOverflowIcon,
     } = data.props;
-    const { renderedOverflowItems, renderedItems } = data;
 
-    const contextualItems = renderedOverflowItems.map(
+    const contextualItems = data.renderedOverflowItems.map(
       (item): IContextualMenuItem => {
         const isActionable = !!(item.onClick || item.href);
         return {
           name: item.text,
           key: item.key,
-          onClick: item.onClick ? this._onBreadcrumbClicked.bind(this, item) : null,
+          onClick: item.onClick ? onBreadcrumbClicked.bind(ref, item) : null,
```

Comment:
Probably better to just do it this way since `this` isn't important.
```suggestion
          onClick: item.onClick ? ev => onBreadcrumbClicked(ev, item) : null,
```